barq/main.py

```diff
@@ -122,52 +132,18 @@ def start():
     logger.addHandler(ch)
     logger.error('calling start')
 
-    myargs = dict(args.grouped)
-    if '--help' in myargs or '-h' in myargs:
-        help = """
-        barq framework options:
-        -h --help         - This menu
-        -k --keyid        - The AWS access key id
-        -s --secretkey    - The AWS secret access key. (Needs --keyid, mandatory)
-        -r --region       - The default region to use. (Needs --keyid)
-        -t --token        - The AWS session token to use. (Needs --keyid, optional)
-        """
-        print(help)
-        exit(0)
-    if '--keyid' in myargs or '-k' in myargs:
-        try:
-            aws_access_key_id = myargs['--keyid'][0]
-        except:
-            aws_access_key_id = myargs['-k'][0]
-        if '--secretkey' not in myargs and '-s' not in myargs:
-            puts(color("[!] using --secretkey is mandatory with --keyid"))
-            exit()
-        try:
-            aws_secret_access_key = myargs['--secretkey'][0]
-        except:
-            aws_secret_access_key = myargs['-s'][0]
-        if '--region' not in myargs and '-r' not in myargs:
-            puts(color("[!] using --region is mandatory with --keyid"))
-            exit()
-        try:
-            region_name = myargs['--region'][0]
-        except:
-            region_name = myargs['-r'][0]
-        if '--token' in myargs or '-t' in myargs:
-            try:
-                aws_session_token = myargs['--token'][0]
-            except:
-                aws_session_token = myargs['-t'][0]
-        else:
-            aws_session_token = ''
-
```

Comment:
The goal here is to accept arguments and assign them to global variables and establish new amazon session. If arguments are empty the user can later setup new profile.

I replaced terminal args with config file to contain all keys not limited to only amazon and simplified the logic to one if statement.
```yml
# config file
amazon:
  id: "value"
  key: "value"
  region: "value"
  token: "value"

# future if needed
digital_ocean:
  id: "value"
...
```
---
```python
# unpack secret keys from config file
aws_access_key_id , aws_secret_access_key, region_name, aws_session_token = get_secret()
# if values are empty, print warning...
# if not empty establish a new amazon session
if not aws_access_key_id or aws_secret_access_key or region_name:
    puts(color('[!] You haven\'t set your AWS credentials yet. Run command setprofile to set them'))
else:
    set_aws_creds_inline(aws_access_key_id, aws_secret_access_key, region_name, aws_session_token)
```